### DISCUSSION TITLE
Add laravel-debugbar to https://demo.filamentphp.com/

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#9637] [Ideas]
Having that will help everyone know what is the "norm". My backend is getting 60+MB per page load and I'm not sure if that is normal. I would be more than happy to PR.

Thank you.

**Comment by sinnbeck** [comment]
Why not just set it up locally and see? Here is the guide
https://github.com/filamentphp/demo#installation


It comes with debugbar but only for local development
https://github.com/filamentphp/demo/blob/main/composer.json#L26

**Reply by bilogic** [reply]
Hi, sure, but that doesn't eliminate my PHP configuration issues. I mean, since the decision was already taken to put up a demo site online, might as well take 1 extra step. :)

**Reply by ModestasV** [reply]
This makes no sense, as too many sensitive details can leak. You should never have debugbar in staging/production.

Imagine they add the debugbar, it leaks the database logins, and someone potentially gets in. That's a no-go situation.

**Reply by bilogic** [reply]
ok, I did not consider the security aspects, so what is the norm for memory usage in a filament resource?

**Reply by ModestasV** [reply]
It really depends. I've seen in the demo ~40MB usage constantly, but in some projects I did was 100+. It all depends on how many models you are loading and so on

**Reply by bilogic** [reply]
![image](https://github.com/filamentphp/filament/assets/946010/175f63d0-873e-461e-9b18-0d9555d3f4ba)

mine's a very reasonable 2 models and 6 queries, but 65MB

**Reply by ModestasV** [reply]
Well, your load time is also big. So, that's probably specific to your PC configuration locally.

**Reply by bilogic** [reply]
![image](https://github.com/filamentphp/filament/assets/946010/4785f990-4f31-438a-90eb-2b4af4356136)


I restarted my PHP FPM and got 9MB for the same page. I suppose no one is really monitoring each requests memory usage...
